### ISSUE TITLE
"// Only change code above this line"  in the right place?

### ISSUE THREAD

**Issue by henriquegeremia** [FreeCodeCamp/FreeCodeCamp#3363]
The comment: "// Only change code above this line." is in the right place?

Challenge [Waypoint: Use Bracket Notation to Find the First Character in a String](http://www.freecodecamp.com/challenges/waypoint-use-bracket-notation-to-find-the-first-character-in-a-string) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.93 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
var firstLetterOfFirstName = "";
var firstLetterOfLastName = "";

var firstName = "Ada";

firstLetterOfFirstName = firstName[0];

var lastName = "Lovelace";

firstLetterOfLastName = lastName;


// Only change code above this line.
// We use this function to show you the value of your variable in your output box.
// You'll learn about functions soon.
(function(v){return v;})(firstLetterOfLastName);

```

**Comment by bugron** [comment]
@henriquegeremia thanks for posting this issue!
Yes it is. You should not change `(function(v){return v;})(firstLetterOfLastName);` because it is used to test your code. You should only change the code which is above that line.
Closing this issue. Feel free to reopen if you still have questions, thanks.

>***
>Please review the [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.
>***

**Comment by henriquegeremia** [comment]
I'm so sorry. 
I was used to reading the same sentence with the word "below", from previous exercises.
So I've got used to the pattern and when I saw the sentence again I didn't read all the words and presume that it was below.
Sorry again for taking your time with my mistake.

But from a design perspective. We know that people, in general, do a quick scan searching for patterns when they're reading.

Maybe rearranging the sentence differently for the below and the above case, in a way that changes the landscape of the sentence, could avoid this kind of mistakes.
